testsuite/integration-arquillian/tests/other/webauthn/pom.xml

```diff
@@ -23,11 +23,6 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>org.keycloak.testsuite</groupId>
-            <artifactId>integration-arquillian-tests-base-ui</artifactId>
```

Comment:
I wasn't entirely sure if this artifact should be removed, but I don't see it referenced anywhere else.